randovania/data/json_data/prime3.txt

```diff
@@ -1927,21 +2270,25 @@ Asset id: 13296637487518400273
 > Above Ground; Heals? False
   > Dock to Temple Access
       Trivial
+  > Dock to Temple Reservoir
+      After Rundas
+  > Pickup (Ice Missiles)
+      After Rundas
   > Pickup (Missile Expansion)
       Suit Type ≥ 4 and Shoot Phazite (X-Ray and Nova)
-  > Below Ground
-      Grapple Lasso and Ship Missile
   > Event - Rundas
       Grapple Lasso
+  > Event - Turret
+      Grapple Lasso and Ship Missile and After Hangar Bay Ship Upgrade
 
-> Below Ground; Heals? False
-  > Dock to Temple Reservoir
-      Trivial
+> Event - Rundas; Heals? False
+  * Event Rundas
   > Above Ground
       Trivial
 
-> Event - Rundas; Heals? False
-  > Pickup (Ice Missiles)
+> Event - Turret; Heals? False
+  * Event Temple of Bryyo Turret
+  > Above Ground
       Trivial
 
 ----------------
```

Comment:
Can Temple Hall be traversed with just SA? If so, change the requirement to be ice missile or sa

Reply:
I haven't tried it myself, and based this off of a few LPs i watched, but if it's possible, it's likely kinda difficult. The room curves quite a bit.